cookiecutter/wrapper-template/{{cookiecutter.wrapper_id}}/{{cookiecutter.library_name}}/main.py

```diff
@@ -0,0 +1,112 @@
+"""{{ cookiecutter.extension_name }} cli entrypoint."""
+
+import os
+import sys
+from typing import List
+
+import structlog
+import typer
+
+from {{ cookiecutter.library_name }}.extension import {{ cookiecutter.extension_name }}
+from meltano.edk.extension import DescribeFormat
+from meltano.edk.logging import default_logging_config, parse_log_level
+
+APP_NAME: str = "{{ cookiecutter.extension_name }}"
```

Comment:
Because this is a literal, the type hint is not necessary.

```suggestion
APP_NAME = "{{ cookiecutter.extension_name }}"
```